web/wp-content/themes/dctx/template-no-title.php

```diff
@@ -0,0 +1,29 @@
+<?php 
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-mz3t3o.php, but found phpcs-scan-mz3T3O.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).